src/ray/gcs/gcs_server/gcs_job_manager.h

```diff
@@ -37,9 +37,10 @@ namespace ray {
 namespace gcs {
 
 // Please keep this in sync with the definition in ray_constants.py.
-const std::string kRayInternalNamespacePrefix = "_ray_internal_";
+const std::string kRayInternalNamespacePrefix = "_ray_internal_";  // NOLINT
```

Comment:
why nolint here?

Reply:
I tried it locally, it errors
```sh
src/ray/util/array.h:49:  For a static/global string constant, use a C style string instead: "const char kRayInternalNamespacePrefix[]".  [runtime/string] [4]
```

Reply:
I'm ok with suppress tidy warning here and L44, because [here](https://github.com/ray-project/ray/blob/ec256d6dad3ef5580fa20ab239b1bd488386e657/src/ray/gcs/gcs_server/gcs_job_manager.h#L45-L47) performs concatenation which requires a string anyway.

Reply:
if better, we should use `inline` for constants in header file, otherwise it's UB.